frontend/app/.server/routes/mappers/benefit-renewal.state.mapper.ts

```diff
@@ -231,27 +236,14 @@ export class DefaultBenefitRenewalStateMapper implements BenefitRenewalStateMapp
   }
 
   mapProtectedRenewStateToProtectedBenefitRenewalDto(
-    { addressInformation, children, confirmDentalBenefits, contactInformation, dentalBenefits, dentalInsurance, hasAddressChanged, hasMaritalStatusChanged, maritalStatus, partnerInformation, clientApplication }: ProtectedRenewState,
+    { addressInformation, children, contactInformation, dentalBenefits, dentalInsurance, maritalStatus, partnerInformation, clientApplication }: ProtectedRenewState,
     userId: string,
   ): ProtectedBenefitRenewalDto {
-    const hasEmailChanged = contactInformation.isNewOrUpdatedEmail;
-    if (hasEmailChanged === undefined) {
-      throw Error('Expected hasEmailChanged to be defined');
-    }
-
-    const hasPhoneChanged = contactInformation.isNewOrUpdatedPhoneNumber;
-    if (hasPhoneChanged === undefined) {
-      throw Error('Expected hasPhoneChanged to be defined');
-    }
-
-    const hasFederalBenefitsChanged = confirmDentalBenefits.federalBenefitsChanged;
-    const hasProvincialTerritorialBenefitsChanged = confirmDentalBenefits.provincialTerritorialBenefitsChanged;
-
     return {
       ...clientApplication,
       applicantInformation: this.toApplicantInformation({
         existingApplicantInformation: clientApplication.applicantInformation,
-        hasMaritalStatusChanged,
+        hasMaritalStatusChanged: false,
```

Comment:
The current `toApplicantInformation(..)` implementation will return the existing applicant info on file if we default `hasMaritalStatusChanged` to `false`, meaning the info will not be updated with the user's specified marital status.

We should check if the `maritalStatus` is defined, meaning the user has changed their marital status, then pass the result to `toApplicantInformation(..)`.

Same thing for `hasEmailChanged`, `hasAddressChanged`, etc. below.